translation-web/src/main/java/eu/europeana/api/translation/TranslationApp.java

```diff
@@ -37,6 +39,7 @@
 public class TranslationApp extends SpringBootServletInitializer {
 
   private static final Logger logger = LogManager.getLogger(TranslationApp.class);
```

Comment:
Rename logger, it seems logger is an already existing field in SpringBootServletInitializer class

Reply:
I'll use the existing logger and change the methods from static to class instance, meaning, we have to move their invocation from main to run method